pre-docs/bolt_configure_orchestrator.md

```diff
@@ -17,19 +17,19 @@ API.
 
 To set up Bolt to use the orchestrator API you must do the following:
 
-- Enable Bolt actions in the PE environment.
+- Install the Bolt module in a PE environment.
 - Set PE RBAC permissions for Bolt tasks.
 - Adjust the orchestrator configuration files, as needed
 - View available tasks
 
 
 ## Enable Bolt actions in the PE environment
 
-Install Bolt source code as a module named 'bolt' in the Puppet code used in
+Bolt uses a task to execute commands, upload files, and run scripts over pcp. To install this this task
```

Comment:
Redundant `this`.

Reply:
Is it clearer to talk about pcp or Orchestrator? I usually prefer to say Orchestrator.